src/main.cpp

```diff
@@ -67,10 +68,12 @@ namespace rhost {
             idle_timeout("rhost-idle-timeout", po::value<std::chrono::seconds::rep>(), (
                 "Shut down the host if it is idle for the specified duration in seconds. "
                 "If " + rdata.long_name() + " was specified, save workspace before exiting."
-                ).c_str());
+                ).c_str()),
+            suppress_ui("suppress-ui", new po::untyped_value(true),
```

Comment:
I believe you can just omit the value here, and only have the name and the docstring.